be/src/vec/sink/vmysql_result_writer.cpp

```diff
@@ -367,6 +367,14 @@ int VMysqlResultWriter::_add_one_cell(const ColumnPtr& column_ptr, size_t row_id
         char buf[64];
         char* pos = datev2.to_string(buf);
         return buffer.push_string(buf, pos - buf - 1);
+    } else if (which.is_date_time_v2()) {
+        auto& column_vector = assert_cast<const ColumnVector<UInt64>&>(*column);
+        auto value = column_vector[row_idx].get<UInt64>();
+        DateV2Value<DateTimeV2ValueType> datetimev2;
```

Comment:
Use `DateV2Value<DateTimeV2ValueType> datetimev2 = binary_cast<UInt64, DateV2Value<DateTimeV2ValueType>>(value)`.

Could you also change datev2 to this method ?

Reply:
sure, will do.